decidim-core/decidim-core.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |s|
   s.add_dependency "date_validator", "~> 0.9.0"
   s.add_dependency "devise", "~> 4.7"
   s.add_dependency "devise-i18n", "~> 1.2"
+  s.add_dependency "diffy", "~> 3.3"
```

Comment:
why do you need diffy in core? it's because amendable is in core?

Reply:
Yes I think so